src/main/scala/mesosphere/mesos/TaskGroupBuilder.scala

```diff
@@ -28,6 +28,8 @@ object TaskGroupBuilder {
     ).asJava)
     .build
 
+  val ephemeralVolPathPrefix = "volumes/"
```

Comment:
The volume is created "somewhere" and mounted into the containers at the defined path.
So the "somewhere" place does not really matter or?
Your example uses relative paths for the volume mounts - I guess absolute paths would be possible as well?